async-development/API.js

```diff
@@ -0,0 +1,14 @@
+let btn = document.getElementById('btn');
+let form = document.getElementById('form');
+form.addEventListener('submit', function() {
+    let city = document.getElementById('input').value;
```

Comment:
Didn't you notice, that it doesn't work on the first search click? 
That's because of you forget `event.preventDefault()`